beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1937,6 +1937,22 @@ pub static LIGHT_CLIENT_SERVER_CACHE_PREV_BLOCK_CACHE_MISS: LazyLock<Result<IntC
         )
     });
 
+pub static LIGHT_CLIENT_SERVER_CACHE_PROCESSING_REQUESTS: LazyLock<Result<IntCounter>> =
+    LazyLock::new(|| {
+        try_create_int_counter(
+            "light_client_server_cache_processing_requests",
```

Comment:
Small nit to prefix the metric with `beacon_` for consistency with other beacon chain metrics and to make it easier to distinguish BN metrics in Grafana queries:

```suggestion
            "beacon_light_client_server_cache_processing_requests",
```